src/transformers/modeling_bart.py

```diff
@@ -1109,7 +1124,7 @@ def _reorder_cache(past, beam_idx):
     def get_encoder(self):
         return self.model.encoder
 
-    def get_output_embeddings(self):
+    def get_output_embeddings(self) -> nn.Linear:
```

Comment:
Still think this should not be generated on the fly ;-) As discussed earlier @sshleifer

Reply:
Should maybe be done in a different PR